sdk/playwrighttesting/create-microsoft-playwright-testing/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@azure/create-microsoft-playwright-testing",
+  "version": "1.0.0-beta.1",
+  "description": "Package to setup @azure/microsoft-playwright-testing",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/playwrighttesting/create-microsoft-playwright-testing/README.md",
+  "sdk-type": "client",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "keywords": [
+    "azure",
+    "cloud",
+    "playwright",
+    "playwright service",
+    "reporting",
+    "playwright testing"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "bin": {
+    "init": "dist/index.js"
+  },
+  "files": [
+    "dist",
+    "README.md",
+    "LICENSE",
+    "CHANGELOG.md"
+  ],
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build": "npm run clean && tsc -p . && dev-tool run bundle && npm run extract-api",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf --glob dist *.log dist-test temp types *.tgz *.xml samples/**/test-results/",
+    "execute:samples": "echo skipped",
+    "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json CHANGELOG.md README.md src --ext .ts,.javascript,.js --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json CHANGELOG.md README.md src --ext .ts,.javascript,.js",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "echo skipped",
+    "unit-test:node": "dev-tool run test:node-ts-input --no-test-proxy -- --timeout 12000 'test/**/*.spec.ts' --exit",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "extract-api": "echo skipped"
+  },
+  "dependencies": {
+    "colors": "^1.4.0",
```

Comment:
Please do not include 3rd party runtime dependencies without archboard review, see [the guidelines on dependencies](https://azure.github.io/azure-sdk/typescript_implementation.html#ts-dependencies) for more information.

I get the appeal and we may consider `commander` given its widespread usage but there are many issues with bundling 3rd party dependencies that need to be vetted first

Reply:
Thank you, and we discussed this internally and are ok approving `prompts` as a runtime dependency in the `create-microsoft-playwright-testing` (initializer) package 👍